iot_verifier/src/settings.rs

```diff
@@ -37,7 +37,7 @@ pub struct Settings {
     pub reward_offset_minutes: i64,
     #[serde(default = "default_max_witnesses_per_poc")]
     pub max_witnesses_per_poc: u64,
-    /// The cadence at which hotspots are permitted to beacon (in hours)
+    /// The cadence at which hotspots are permitted to beacon (in seconds)
```

Comment:
if this should always be a factor of 24 then add a check to the accessor that validates it evenly divides by 24 or else panics before returning the i64 converted to a duration

Reply:
addressed here: b62179d0dbc7d16b8f4bd2321ab20a7dc8f80454